### DISCUSSION TITLE
Time setting of Clients not being local or Bios dependable

### DISCUSSION THREAD

**Discussion by perlland** [ltsp/ltsp#504] [Q&A]
I am having LTSP server configured Ubuntu Mate 20.04. After installation and updating I make LTSP server Offline for internal LAN purpose but my LTSP Clients are not getting server timing, but they are showing wrong timing. The Date showing in the clients are right. but not the time. How could I rectify this being server and client machines are offline and can't be connected to online. Kindly help in this regard. The earlier response is deeply solicited. Thanking You in advance

**Comment by alkisg** [comment]
I didn't understand the question. LTSP doesn't automatically install nor it configures time services.

Do you want to install a time server in the LTSP server, and tell the clients to use it for time synchronization?
This isn't specific to LTSP, you may google for any tutorial about it.
